### DISCUSSION TITLE
Error "no datatable request detected"

### DISCUSSION THREAD

**Discussion by rmatos10** [hermawanramadhan/CodeIgniter4-DataTables#27] [General]
Even following all the correct steps in the installation manual, the error "no data request detected" appears, does anyone know why? I'm using CodeIgniter 4.2.6. I also tested in version 4.2.0 the problem follows

**Comment by rmatos10** [comment]
Sorted out. Problem was related to POST Format coming from "Angular DataTables - l-lin" plugin. I switched to using jQuery's Official DataTable in Angular 12 and it worked great

**Comment by hermawanramadhan** [comment]
pleased to hear it!

**Comment by theodorichas** [comment]
Hi, I'm having the same problem where it showed me     "error": "no datatable request detected"
I followed all of your steps, how can I fix this?

**Reply by newsepo** [reply]
I too am having the same problem.
